qa/abstracts/src/main/java/io/camunda/zeebe/process/test/qa/abstracts/util/Utilities.java

```diff
@@ -188,12 +192,24 @@ public static void increaseTime(final ZeebeTestEngine engine, final Duration dur
       throws InterruptedException {
     try {
       waitForIdleState(engine, Duration.ofSeconds(1));
-      engine.increaseTime(duration);
+    } catch (final TimeoutException e) {
```

Comment:
🔧 Let's add a detailed description why we added these different try catch errors to this method. Also, let's mention that these changes should be reverted when we understand the flakiness of the `UnhappyPathTests.testHasNotExpiredFailure`.